.storybook/main.js

```diff
@@ -12,6 +12,17 @@ module.exports = {
     reactDocgen: false,
   },
   webpackFinal: async (config) => {
+    // Default rule for images /\.(svg|ico|jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|cur|ani|pdf)(\?.*)?$/
+    const fileLoaderRule = config.module.rules.find(rule => rule.test && rule.test.test && rule.test.test('.svg'));
```

Comment:
nit: should this be rules instead of rule?

Reply:
But `fileLoaderRule` actually refers to only one rule that handles `.svg` and uses `file-loader` out of the webpack rules that storybook uses by default. This rule in the [default webpack config](https://github.com/storybookjs/storybook/blob/next/lib/builder-webpack4/src/preview/base-webpack.config.ts) is as follows:

```
{
          test: /\.(svg|ico|jpg|jpeg|png|apng|gif|eot|otf|webp|ttf|woff|woff2|cur|ani|pdf)(\?.*)?$/,
          loader: require.resolve('file-loader'),
          options: {
            esModule: false,
            name: isProd
              ? 'static/media/[name].[contenthash:8].[ext]'
              : 'static/media/[path][name].[ext]',
          },
},
```
So `fileLoaderRule` refers to that rule that we will later modify to exclude `.svg` files and add our own loader.